models/README.md

```diff
@@ -6,13 +6,13 @@ A split is made between [languages](./meta), and [instances](./instance) of thos
 
 ## Languages
 
-Serializations of languages, i.e. instances of the `Language` (meta-)concept from LIonCore/M3.
+Serializations of languages, i.e. instances of the `Language` (meta-)concept from LionCore/M3.
```

Comment:
Is `LionCore/M3` (with a slash) really the correct spelling?

Reply:
I didn't read the context, just wanted to make sure it's not a missed instance. I don't have a preference.